alonet/common/pl_helpers.py

```diff
@@ -71,6 +71,7 @@ def add_argparse_args(parent_parser, add_pl_args=True, mode="training"):
     parser.add_argument("--project_run_id", type=str, help="Project related with the run ID to load")
     parser.add_argument("--expe_name", type=str, default=None, help="expe_name to be logged in wandb")
     parser.add_argument("--no_suffix", action="store_true", help="do not add date suffix to expe_name")
+    parser.add_argument("--nostrict", action="store_false", help="load from checkpoint to run a model with different weights (default True)")
```

Comment:
Usually if one use --nostrict the expected behavior is : Do not strictly load the weights (store true on nostrict)
But here --nostrict set nostrict to false. It seem to me to be counter intuitive.

Reply:
now if --nostrict is specified (store True and False by default) it is possible to load a checkpoint with different keys in the load_state_dict. BUT it is still not possible for a checkpoint with same keys but different values (weights dimensions).